vaultpress.php

```diff
@@ -3046,12 +3046,9 @@ function register_via_jetpack( $already_purchased = false ) {
 
 if ( isset( $_GET['vaultpress'] ) && $_GET['vaultpress'] ) {
 	if ( !function_exists( 'wp_magic_quotes' ) ) {
-		// If already slashed, strip.
-		if ( get_magic_quotes_gpc() ) {
-			$_GET    = stripslashes_deep( $_GET    );
-			$_POST   = stripslashes_deep( $_POST   );
-			$_COOKIE = stripslashes_deep( $_COOKIE );
-		}
+		$_GET    = stripslashes_deep( $_GET    );
+		$_POST   = stripslashes_deep( $_POST   );
+		$_COOKIE = stripslashes_deep( $_COOKIE );
 
```

Comment:
```suggestion
```

Removing this section as per the discussion above.

Reply:
https://core.trac.wordpress.org/browser/tags/5.7.1/src/wp-includes/load.php#L1049